storybook/package.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "storybook",
+  "version": "0.0.1",
+  "type": "module",
+  "license": "Apache-2.0",
+  "scripts": {
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build"
+  },
+  "peerDependencies": {
+    "svelte": "^4.0.0",
+    "svelte-fa": "^4.0.0"
+  },
+  "dependencies": {
```

Comment:
need to check that all these dependencies are the latest one to prevent tons of dependabot requests

Reply:
Should be good